artichoke-core/src/class_registry.rs

```diff
@@ -0,0 +1,40 @@
+use std::any::Any;
```

Comment:
```suggestion
use core::any::Any;
```

`artichoke-core` can be built without `std` and `Any` is available in `core`, so let's import it from there.